eco.Rmd

```diff
@@ -25,42 +25,47 @@ finally, point to books on ecological modeling especially emphasizing Alain Zuur
 
 Fog oases are one of the most fascinating vegetation formations I have ever encountered. 
 These formations, locally termed *lomas*, develop on mountains along the coastal deserts of Peru and Chile.
-The desert's extreme harsh conditions and its remoteness spawned a truly unique and endemic flora and fauna on these mountains.
-The annual rain input is less than 30-50 mm on average.
-Still, plants survive this extreme aridity by combing out fog which develops below the temperature inversion caused by the cold Humboldt current in austral winter.
-Every once in a few years, El Ni{\~n}o brings torrential rainfalls to this sun-baked environment causing on the one hand the desert to bloom, and on the other hand giving tree seedlings the chance to develop roots long enough to survive the following aridity.
```

Comment:
Markdown allows you to use such symbols directly without using LaTeX.

Reply:
good to know 👍